.gitignore

```diff
@@ -0,0 +1,5 @@
+.vscode
+**/bin
+**/obj
+src/Properties
+appsettings.Development.json
```

Comment:
Add an empty line here (GitHub doesn't seem to like it when files don't end in empty lines :stuck_out_tongue: )
![image](https://user-images.githubusercontent.com/16925524/45169930-0bdd4100-b1d5-11e8-8e95-ec7b5abb9b62.png)